stake-pool/cli/src/main.rs

```diff
@@ -797,8 +801,8 @@ fn pick_withdraw_accounts(
     // Not enough stake to withdraw the specified amount
     if remaining_amount > 0 {
         return Err(format!(
-            "No stake accounts found in this pool with enough balance to withdraw {} SOL.",
-            lamports_to_sol(amount)
+            "No stake accounts found in this pool with enough balance to withdraw {} pool tokens.",
+            lamports_to_sol(pool_amount)
```

Comment:
Little weird to use `lamports_to_sol()` on pool tokens here, could `spl_token::amount_to_ui_amount()` be used instead?